### PR TITLE
Add files via upload

### PR DESCRIPTION
**Pull request by JamesMcGowan123**
After much toil and trouble, the early version of our witch class is finally here and available to you, our beloved patrons!! A lot of care and deliberation went into developing the class for Erika to use, and we’ve been so eager to share it with you all and hear what you think.

We have to stress that this is an early draft of the class mechanics—it hasn’t been thoroughly playtested (you’ve basically heard the extent of it on the podcast), so GMs should be fully aware of that before allowing it to be used in their games.

Because of this, we’ll be seeking feedback from the Patreon in a month or two, once people have had some time to play the class and test it out. We don’t expect feedback, but if you decide to try it out and are willing to share some thoughts, we’d love to hear from you, so keep an eye out!

Big thanks to everyone who helped us put this together, including designers Mazey Veselak, Brandes Stoddard, and Hannah Rose; layout artist Ruby Lavin; illustrators Corey Brickley, Tucker Donovan, and Taylor Moore; and original character illustrator Lorena Lammer.

### CODE REVIEW

File: class/Worlds Beyond Number; The Witch.json [hunk 1]
```diff
@@ -0,0 +1,2767 @@
...
+				},
+				"Ability Score Improvement|Witch|WitchWBN|16",
+                "Willful Walls|Witch|WitchWBN|18",
+				"Ability score Improvement|Witch|WitchWBN|19",
+				"True Craft|Witch|WitchWBN|20"
+			],
+			"subclassTitle": "Covens",
+			"classSpells": [
+				"Acid Splash",
+				"Cackle|WitchWBNSpells",
```
Comment (revilowaldow) [inline class/Worlds Beyond Number; The Witch.json:457]:
```suggestion
				"Cackle|WitchWBN",
```

File: class/Worlds Beyond Number; The Witch.json [hunk 2]
```diff
@@ -0,0 +1,2767 @@
...
+				"Fog Cloud",
+				"Heroism",
+				"Tasha's Hideous Laughter",
+				"Inflict Wounds",
+				"Protection from Evil and Good",
+				"Shield",
+				"Silent Image",
+				"Sleep",
+				"Speak with Animals",
+				"Witch's Grasp|WitchWBNSpells",
```
Comment (revilowaldow) [inline class/Worlds Beyond Number; The Witch.json:485]:
```suggestion
				"Witch's Grasp|WitchWBN",
```

File: class/Worlds Beyond Number; The Witch.json [hunk 3]
```diff
@@ -0,0 +1,2767 @@
...
+				"Aid",
+				"Animal Messenger",
+				"Arcane Lock",
+				"Augury",
+				"Blindness/Deafness",
+				"Breath of Belladonna|WitchWBNSpells",
+				"Cleansing Crystal|WitchWBNSpells",
```
Comment (revilowaldow) [inline class/Worlds Beyond Number; The Witch.json:492]:
```suggestion
				"Breath of Belladonna|WitchWBN",
				"Cleansing Crystal|WitchWBN",
```

**Review by revilowaldow — CHANGES_REQUESTED** [review]
Actually never mind there's loads more than I anticipated.
Everywhere you use  `WitchWBNSpells`, switch it for `WitchWBN`

File: class/Worlds Beyond Number; The Witch.json [hunk 4]
```diff
@@ -0,0 +1,2746 @@
...
+				{
+					"type": "list",
+					"items": [
+						"{@item Elixir of health}",
+						"{@item Potion of animal friendship}",
+						"{@item Potion of fire breath}",
+						"{@item Potion of growth}",
+						"{@item Potion of healing}",
+						"{@item Potion of mind reading}",
+						"{@item Potion of resistance} (you decide the type of resistance it grants)}",
```
Comment (revilowaldow) [inline class/Worlds Beyond Number; The Witch.json:1481]:
```suggestion
						"{@item Potion of resistance} (you decide the type of resistance it grants)",
```

Comment (JamesMcGowan123) [reply class/Worlds Beyond Number; The Witch.json]: I have deleted the }

**Review by TheGiddyLimit — APPROVED** [review]

**Pull request merged**